activiti-core/activiti-api-impl/activiti-api-runtime-shared-impl/src/test/java/org/activiti/runtime/api/event/impl/ToVariableCreatedConverterTest.java

```diff
@@ -60,4 +61,32 @@ void should_convertToVariableCreatedEvent() {
         assertThat(actualValue).isSameAs(value);
     }
 
+    @Test
+    void should_convertToVariableCreatedEvent_withEmptyValue_when_variableIsEphemeral() {
+        ActivitiVariableEventImpl internalEvent = new ActivitiVariableEventImpl(ActivitiEventType.VARIABLE_CREATED);
+        internalEvent.setVariableName("variableName");
+        internalEvent.setProcessInstanceId("processInstanceId");
+        internalEvent.setProcessDefinitionId("processDefinitionId");
+        internalEvent.setTaskId("taskId");
+        VariableType variableType = new StringType(100);
+        internalEvent.setVariableType(variableType);
+        Object value = mock(Object.class);
```

Comment:
In this case I think instead of mocking it's easier any value of your choice, a String value for instance
```
Object value = "any";
```

Or event set it directly:
`internalEvent.setVariableValue("any")`;